docs/sql-ref-ansi-compliance.md

```diff
@@ -64,15 +64,15 @@ On the other hand, `INSERT INTO` syntax throws an analysis exception when the AN
 Currently, the ANSI mode affects explicit casting and assignment casting only.
 In future releases, the behaviour of type coercion might change along with the other two type conversion rules.
 
-{% highlight sql %}
+```sql
```

Comment:
Seems OK, is there any behavior difference?
I'm on the fence about whether it's worth changing across all files.

Reply:
Both of them highlight SQL keywords the same way. The only difference I noticed is that ```sql doesn't indent the code block: 
<img width="503" alt="Screen Shot 2020-05-05 at 8 34 07 AM" src="https://user-images.githubusercontent.com/13592258/81085006-76171f00-8eab-11ea-8d3d-acb82e26fbca.png">


but {% hightlight sql %} indents
<img width="566" alt="Screen Shot 2020-05-05 at 8 35 18 AM" src="https://user-images.githubusercontent.com/13592258/81085021-7ca59680-8eab-11ea-83fb-732fb92eae24.png">